Gruntfile.js

```diff
@@ -1,81 +1,84 @@
 module.exports = function(grunt) {
-  grunt.loadNpmTasks('grunt-browserify');
 
-  require('matchdep').filterDev('grunt-*').forEach(grunt.loadNpmTasks);
+    grunt.loadNpmTasks('grunt-browserify');
```

Comment:
Uh oh - were these commits supposed to make it into here? Just checking!

Reply:
Ah, i see here! all good then! https://github.com/publiclab/PublicLab.Editor/pull/566#issuecomment-660536609